### ISSUE TITLE
Needs better documentation

### ISSUE THREAD

**Issue by dundeemt** [anjishnu/ask-alexa-pykit#18]
The README.md is ok general documentation if you already have experience with the package. However, it would be great if there was a walk-through where you re-create alexa-skills-kit-color-expert example but using ask-alexa-pykit.

Since color-expert / what's my fav color is the "hello world" of Alexa and is well documented on aws, it would be an outstanding example to use to show how it would be accomplished using ask-alexa-pykit.  It was my goal to recreate it using your toolkit for the reason I outlined above.

**Comment by anjishnu** [comment]
Good idea. I don't have time to look into it right now - If anyone get's around to doing it - feel free to open a pull request.
